test/rbs/collection/config_test.rb

```diff
@@ -212,66 +233,16 @@ def test_generate_lock_from_collection_repository_ignoring
     end
   end
 
-  def test_generate_lock_from_collection_repository_specified
```

Comment:
We cannot delete this test because a `default` gem may be required without declared in `Gemfile`...

Reply:
This should be ok. If a *non-default* gem is not included in the Bundler context, it cannot be `require`-ed nor `Gem::Specification.find_by_name` cannot find the gem.

If a *non-default* gem is implicitly loaded, it should be included in the `gems` section of `rbs_collection.yaml` or `dependencies:` section of `manifest.yaml` of a gem.